ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -98,6 +98,12 @@
     reason="Tests are not running on AWS deployed cluster"
 )
 
+rh_internal_lab_required = pytest.mark.skipif(
+    (config.ENV_DATA['platform'].lower() == 'aws'
```

Comment:
There is a problem with this approach. This is a static value and for baremetal deployments, config shows up as `AWS` as that is the default value. Maybe, we should also default `config.ENV_DATA['platform']` to 'None'?

Reply:
This mark will skip if its aws or gcloud, so anything else should work, for baremetal we should fix it to use 'baremetal' instead of 'none'

Reply:
We definitely don't want to default to `None` as the expected default behavior is to deploy on aws. We should have a "platform" for baremetal that is set in the baremetal configuration file. This way the platform does not appear as configured for aws when it is baremetal.

Reply:
As long as we don't have Baremetal deployment automated, `config.ENV_DATA['platform']` will be set to `AWS` isn't it?

Reply:
@keesturam you can still pass a custom config file through when running tests that changes the platform to 'baremetal' or something other than 'aws'.

Reply:
@clacroix12 Thanks, that clarifies.